rtl/eclrtl/eclrtl.cpp

```diff
@@ -3803,15 +3803,7 @@ hash64_t rtlHash64Data(size32_t len, const void *buf, hash64_t hval)
 
 hash64_t rtlHash64VStr(const char *str, hash64_t hval)
 {
-    const unsigned char *s = (const unsigned char *)str;
-    unsigned char c;
-
-    while ((c = *s++) != 0)
-    {
-        APPLY_FNV64(hval, c);
-    }
-
-    return hval;
+    return rtlHash64Data(rtlTrimVStrLen(str), str, hval);
```

Comment:
Is this just cleaning up the code or have we been incorrectly calculating the hashes of varstrings? The implications of the second scenario may be concerning (and might warrant a separate Jira?)